### DISCUSSION TITLE
Can't verify my phone number and phone number blocked for more than 24 hours

### DISCUSSION THREAD

**Discussion by tmp1999** [vercel/community#5088] [Account Issue] [answered]
### Question

Hi, I have been unable to login into vercel with my GitHub account. I got stuck at the verification stage. I have tried using another phone number to receive the verification code via SMS but I still can't get it.
Every time I get this error ,
Error: The verification phone number has been temporarily blocked. Please wait 12 hours and try again.
Phone number blocked for more than 24 hours.
i sent email address and username, phone number to registration@vercel.com but remains unresolved
Your help will be greatly appreciated.
Thanks

**Comment by swarnava** [comment, marked as answer]
Hi,

Thank you for writing in registration@vercel.com. Our team will reach out about the next steps via email.

**Reply by tmp1999** [reply]
Hi. 
I login with github get error: There is no Vercel account linked with this GitHub account.
with email verify phone number still blocked 12 hours. 
Can u help me?
Thanks.

**Comment by tmp1999** [comment]
Hi there. I try to sign in with Github again but my phone number is still
blocked
When you're done with the problem, please get back to me
Thanks

Vào Th 4, 6 thg 12, 2023 vào lúc 16:26 Swarnava Sengupta <
***@***.***> đã viết:

**Reply by swarnava** [reply]
Hi, this is because our support team hasn't looked at your request yet. Once they triage and send you email, you should be able to access it.

**Comment by tmp1999** [comment]
Ok, thank you.

Vào Th 4, 6 thg 12, 2023 vào lúc 18:27 Swarnava Sengupta <
***@***.***> đã viết:
